content/courses/solana-pay/solana-pay.md

```diff
@@ -586,9 +582,9 @@ async function post(req: NextApiRequest, res: NextApiResponse) {
         let error = err as any
         if (error.message) {
             res.status(200).json({ transaction: "", message: error.message })
-        } else {
-            res.status(500).json({ error: "error creating transaction" })
+            return
```

Comment:
Run prettier over this code. See https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#jsts re: markdown and prettier.

Reply:
Fixed.

<img width="396" alt="Screenshot 2024-08-26 at 22 01 16" src="https://github.com/user-attachments/assets/8af9cb38-04bd-4a93-9a9b-d19090eeec4a">

Reply:
Sorry, there's something you missed in https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#jsts re: markdown and prettier. Quoting:

> Note that while prettier can format Markdown, https://github.com/prettier/prettier/issues/5378 so you'll need to format the code yourself for now. To do this quickly, make a blank deleteme.ts file, paste your code in there, and then save it (allowing prettier to do its work), and paste it back into the document.